R/entities_objects.R

```diff
@@ -0,0 +1,114 @@
+# Contains functions to parse the objects described here: 
+# https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/entities
+
+# <https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/entities#hashtags>
+hashtags <- function(x) {
+  if (NROW(x) == 0) {
+    return(data.frame(text = NA, indices = I(list(NA)), 
+                      stringsAsFactors = FALSE))
+  }
+  i <- indices_vec(x$indices)
+  data.frame(text = x$text, indices = I(i))
+}
+
+
+indices_vec <- function(x) {
+  i <- lapply(x, function(y){matrix(y, ncol = 2, 
+                                    dimnames = list(NULL, 
+                                                    c("start", "end")))})
+  i
+}
+
+# The extended entities is really for media
+# <https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/extended-entities>
+media <- function(x) {
+  df <- data.frame("id" = NA, "id_str" = NA, "indices" = I(list(NA)), 
```

Comment:
Don't need quotes around names?

Reply:
Correct, removed the quotes from other data.frames too